src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const lowerMovieTitle = movie.title.toLowerCase();
+    const lowerMovieDescription = movie.description.toLowerCase();
+    const querySearch = query.toLowerCase().trim();
+
+    return (lowerMovieTitle.includes(querySearch)
```

Comment:
DRY
Move this check `if str includes substr` to the separate function & move this function from the component